compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -216,7 +216,6 @@ class CompilationTests {
   @Test def checkInitGlobal: Unit = {
     implicit val testGroup: TestGroup = TestGroup("checkInitGlobal")
     val options = defaultOptions.and("-Ysafe-init-global", "-Xfatal-warnings")
```

Comment:
We should inline `options` into `compileFilesInDir("tests/init-global/pos", options,`